lib/commands/web-view/set-web-view.ts

```diff
@@ -0,0 +1,45 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+import commandParams = require("../../common/command-params");
+
+export class SetWebViewCommand implements ICommand {
+	constructor(private $webViewService: IWebViewService,
+		private $injector: IInjector,
+		private $errors: IErrors) { }
+	
+	public execute(args: string[]): IFuture<void> {
+		return this.$webViewService.enableWebView(args[0], args[1]);
+	}
+	
+	public canExecute(args: string[]): IFuture<boolean> {
+		return (() => {
+			if(!args[0] || !args[1]) {
+				this.$errors.fail(`Platform and web view name are mandory params.`);
+			}
+			
+			this.$webViewService.validateFrameworkVersion();
+			
+			let supportedWebViews = this.$webViewService.supportedWebViews;
+			
+			// Validate platform
+			let platform = args[0].toLowerCase();
+			let platforms = _.keys(supportedWebViews);
+			if(!_.contains(platforms, platform)) {
+				this.$errors.failWithoutHelp(`Invalid platform. The valid platforms that support web view are: ${platforms.join(", ")}`);
+			}
+			
+			// Validate webView
+			let webView = args[1].toLowerCase()
+			let webViewNames = this.$webViewService.getWebViewNames(platform);
+			if(!_.contains(webViewNames, webView)) {
+				this.$errors.failWithoutHelp(`Invalid web view. The valid webViews for ${platform} platform are: ${webViewNames.join(", ")}`);
```

Comment:
We should standartize on the WebView naming. Current we have web view, webView. My proposal is **WebView**.